src/rpcServerExplorer.ts

```diff
@@ -28,7 +28,14 @@ const TemplateInstructionsSecondRun = 'To update custom RPC servers, edit your J
 
 const MainNetServerListUrl = 'https://api.neoscan.io/api/main_net/v1/get_all_nodes';
 
-const TestNetServerListUrl = 'https://neoscan-testnet.io/api/main_net/v1/get_all_nodes';
+// Retrieved from http://monitor.cityofzion.io/
+const TestNetServerUrls = [
```

Comment:
So with this approach, we only support the hard coded URLs?

Reply:
Yeah, the API for retrieving the TestNet URLs seems to have been down for a long time.  I think this approach is fine though, given the ease of shipping an extension update and the speed at which all clients update to the new extension.